cirq/ops/gate_operation.py

```diff
@@ -189,6 +189,18 @@ def __pow__(self, exponent: Any) -> 'raw_types.Operation':
             return NotImplemented
         return self.with_gate(new_gate)
 
+    def __mul__(self, other):
```

Comment:
Missing type annotations here and on `__rmul__` below.

Reply:
Added `Any`s.